src/applications/hca/components/FormFields/VaMedicalCenter.jsx

```diff
@@ -1,44 +1,83 @@
 import React, { useCallback, useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
 import * as Sentry from '@sentry/browser';
 
 import { VaSelect } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
-import environment from 'platform/utilities/environment';
-import { apiRequest } from 'platform/utilities/api';
-import { focusElement } from 'platform/utilities/ui';
+import constants from 'vets-json-schema/dist/constants.json';
+import { usePrevious } from '~/platform/utilities/react-hooks';
+import environment from '~/platform/utilities/environment';
+import { apiRequest } from '~/platform/utilities/api';
+import { focusElement } from '~/platform/utilities/ui';
+import { STATES_WITHOUT_MEDICAL } from '../../utils/constants';
 import ServerErrorAlert from '../FormAlerts/ServerErrorAlert';
+import { VaMedicalCenterReviewField } from '../FormReview/VaMedicalCenterReviewField';
 
 const apiRequestWithUrl = `${
   environment.API_URL
 }/v0/health_care_applications/facilities?type=health&per_page=1000`;
 
 const VaMedicalCenter = props => {
-  const { formContext, id, onChange, required, value, facilityState } = props;
+  const {
```

Comment:
Just wanted to comment I like having these on new lines like you have done. It makes reading commit history diffs easier for me. 👍

Reply:
Thanks Prettier - the new props exceeded the line length!